src/main/java/com/oltpbenchmark/benchmarks/wikipedia/WikipediaLoader.java

```diff
@@ -101,6 +101,8 @@ public List<LoaderThread> createLoaderThreads() {
             public void load(Connection conn) throws SQLException {
                 Table catalog_tbl = benchmark.getCatalog().getTable(WikipediaConstants.TABLENAME_USER);
 
+                SQLUtil.setIdentityInsert(conn, getDatabaseType(), catalog_tbl, true);
```

Comment:
Basically a SqlServer specific workaround for being able to insert into a table with auto-increment columns.